.github/workflows/lint.yml

```
@@ -20,6 +26,9 @@ jobs:
       contents: read
     runs-on: ubuntu-latest
     steps:
+      - uses: step-security/harden-runner@v1
+        with:
+          egress-policy: audit
```

```suggestion
          ghcr.io:443
          github.com:443
          pkg-containers.githubusercontent.com:443
```
i assume ghcr.io is a github thing?

---

Yes. https://github.com/features/packages